### ISSUE TITLE
Dark/night mode

### ISSUE THREAD

**Issue by WildGenius** [shuowu/yi-note#73]
Would be lovely to have the option to switch to a whitish text on a darkish background. I'm quite used to using Dark Reader or Stylus to darken the web, so the stark white of YiNote is quite the contrast.

**Comment by tokyojerry** [comment]
@shuowu, yes, I also would like to have a dark mode theme as described by @WildGenius
